defaults.go

```diff
@@ -166,8 +167,68 @@ func (dns *Msg) IsEdns0() *OPT {
 // label fits in 63 characters, but there is no length check for the entire
 // string s. I.e.  a domain name longer than 255 characters is considered valid.
```

Comment:
This comment is not true anymore

Reply:
I actually don't think this was ever true.